spec/features/photoes_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'rails_helper'
+
+feature 'Uploading photoes' do
+
+  scenario 'a not logged in user can\'t upload a photo' do
+    visit '/'
+    click_link 'Upload'
+    expect(current_path).to eq '/users/sign_in'
+  end
+
+  context 'when logged in' do
+    background do
+      visit '/'
+      sign_up(username: 'allimac',email: 'camilla@email.com', password: 'pass1234')
```

Comment:
Metrics/LineLength: Line is too long. [83/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)